Makefile.fem

```diff
@@ -1,35 +1,26 @@
 SHELL := /bin/bash
 
-target:
-	@mkdir -p target/certs
-	@mkdir -p target/config
+prepare-fem:
+	@mkdir -p target/fem/certs
+	@chmod 755 target/fem/certs
+	@mkdir -p target/fem/config
+	@chmod 755 target/fem/config
+	@cp -n fem/docker-compose-fem.yml target/fem
 
-docker-fem: target
-	@cp -n docker-compose-fem.yml target
+config: prepare-fem
+	@cp -rn fem/config target/fem
+	@chmod 644 target/fem/config/*.yml
 
-docker-fem-demoapp: target
-	@cp -r demoapp target
-	@cp -n docker-compose-fem-demoapp.yml target
+certs: prepare-fem
+	@openssl req -newkey rsa:4096 -nodes -keyout target/fem/certs/server.key -subj "/C=DE/ST=Sachsen/L=Dresden/O=T-Systems MMS GmbH/CN=monitoring-host" -out target/fem/certs/server.csr
+	@openssl x509 -req -extfile <(printf "subjectAltName=DNS:fem_otel-collector,DNS:localhost,IP:127.0.0.1") -days 365 -in target/fem/certs/server.csr -signkey target/fem/certs/server.key -out target/fem/certs/server.crt
+	@chmod 644 target/fem/certs/server.key target/fem/certs/server.crt
 
-config: target
-	@cp -r config target/
+start: config certs
+	docker-compose --file target/fem/docker-compose-fem.yml up -d
```

Comment:
@ronnyfriedland , I think the definition of the project name is needed now, e.g. add: "--project-name fem".
With the separation into modules it must be ensure that all components run in the same network.
As you know without configure the project name, the name for the container as well as the name for the network were created from the directory name by default. That results into different subnets for the containers. The avoid that we have to set the project name and remove the "fem_"-prefix in all docker-compose files.